src/people.rs

```diff
@@ -1368,7 +1379,7 @@ mod test {
         context
             .get_data_container(PeoplePlugin)
             .unwrap()
-            .get_index_ref(TypeId::of::<T>())
+            .get_index_ref_mut(TypeId::of::<T>())
```

Comment:
I think you don't need the ref_mut here and the original code will work as this function doesn't need the mutability.